cli/install/install.go

```diff
@@ -492,36 +578,40 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 		}
 	} else {
 		log.Infof("Downloading tt...")
-		err = downloadRepo(search.GitRepoTT, ttVersion, path, logFile, installCtx.verbose)
+		if versionFound {
+			err = downloadRepo(search.GitRepoTT, ttVersion, path, logFile, installCtx.verbose)
+		} else {
+			err = downloadRepo(search.GitRepoTT, "master", path, logFile, installCtx.verbose)
+			util.ExecuteCommand("git", installCtx.verbose, logFile, path, "checkout", ttVersion)
```

Comment:
No error check. We should not run `git` command if repo is not cloned.

Reply:
fixed